### PR TITLE
Multidisplay support

### LINKED ISSUE #67: 8-digit seven segment display
Is there a tiny chance to support 8-digit 7-seg display? As we can see, HT16K33 pinout supports 8 digits, but...

### LINKED ISSUE #74: Add Multiple Display Support for 7-Segment, Alphanumeric, and Matrix Displays
Multiple Display Support would essentially "soft-chain" two or more HT16K33 based displays so larger numeric, alphanumeric text, and longer matrix displays could be created. When I say "soft-chain," I mean that the chaining of displays would be done transparently in software because each set of four display units requires a separate HT16K33 controller. It just takes some software magic behind the scenes to make it appear like several of these displays are all one unit.

I would love to add support for multiple 7-segment, alphanumeric, and matrix displays. However, I only have one of the alphanumeric display Feather wings (Green) here. To do this and properly test everything, I would need at least three of each display. I think this would be an _**awesome**_ feature to add to the library, and it _may_ be possible to make it entirely transparent to the user.

### PR DESCRIPTION
**Pull request by makermelissa**
Fixes #74. Fixes #67.

This has been tested on the FeatherWings. I have included examples. Library should function as before with single displays. The main difference is a lot of the hard-coded numbers are now calculated.

### CODE REVIEW

File: adafruit_ht16k33/ht16k33.py [hunk 1]
```diff
@@ -43,13 +43,19 @@ class HT16K33:
     def __init__(
         self,
         i2c: I2C,
-        address: int = 0x70,
+        address: Union[int, Tuple, List] = 0x70,
```
Comment (tekktrik) [inline adafruit_ht16k33/ht16k33.py:46]: You can get away with using `Union[int, Sequence[int]]` here, but that technically also allows for using things like `bytes` and `bytearray` which isn't the intention.  So probably better to just specifythe types within the generics:

```python
        address: Union[int, Tuple[int, ...], List[int]] = 0x70

File: adafruit_ht16k33/ht16k33.py [hunk 2]
```diff
@@ -43,13 +43,19 @@ class HT16K33:
...
         auto_write: bool = True,
         brightness: float = 1.0,
     ) -> None:
-        self.i2c_device = i2c_device.I2CDevice(i2c, address)
+        if isinstance(address, (tuple, list)):
+            self.i2c_device = []
+            for addr in address:
+                self.i2c_device.append(i2c_device.I2CDevice(i2c, addr))
```
Comment (tekktrik) [inline adafruit_ht16k33/ht16k33.py:53]: This is a neat solution.  The only thing that comes to mind is that this technically now means that this library wouldn't work with `adafruit_register` but that doesn't apply here.

**Review by tekktrik — COMMENTED** [review]
Took a look and saw typing updates, so just looked at those, didn't test.  Nice!

File: adafruit_ht16k33/ht16k33.py (hunk 1 above)
Comment (makermelissa) [reply adafruit_ht16k33/ht16k33.py:46]: Thanks. Yeah, I'd prefer to not include bytes or bytearrays at this time.

File: adafruit_ht16k33/ht16k33.py (hunk 2 above)
Comment (makermelissa) [reply adafruit_ht16k33/ht16k33.py:53]: Thanks, yeah.

**Comment by makermelissa** [comment]
Currently this only supports side by side matrices, but it should be easy enough to update for vertically stacked I think.

**Review by dhalbert — APPROVED** [review]
Re-approved after minor change

**Pull request merged**